src/porepy/__init__.py

```diff
@@ -72,8 +72,8 @@
 from porepy.grids.mortar_grid import MortarGrid, BoundaryMortar
 
 # Fractures
-from porepy.fracs.fractures import Fracture, EllipticFracture, FractureNetwork
-from porepy.fracs.meshing import simplex_grid
+from porepy.fracs.fractures import Fracture, EllipticFracture, FractureNetwork3d
```

Comment:
why FractureNetwork2d is in one module and FractureNetwork3d is in the standard fractures module?